### DISCUSSION TITLE
Contacts "sort by" doesn't remember what I choose

### DISCUSSION THREAD

**Discussion by serrq** [tutao/tutanota#4923] [Ideas]
I selected "sort by" last name.

Every time I close and reopen the app I have to repeat the choice. Always.

**Comment by RilkoyMaName** [comment]
Same here. Happens both in desktop and mobile app (Apple devices).
This discussion is quite old -- no one except the two of us having this issue?

**Comment by Markoise** [comment]
It's the same for me. It is a bit annoying.

**Comment by ganthern** [comment]
Hey there :wave:

This is not considered a persistent setting, more like a filter / collation feature on the list itself. It's worth thinking about if it should be, either globally or per contact list.

**Comment by serrq** [comment]
Make it persistent. End.

**Comment by Grayintas** [comment]
Me too, agreed
